generateData/sampleData/generateData.js

```diff
@@ -8,6 +8,7 @@
 */
 
 const data = require('./json.js');
+console.log(data)
```

Comment:
don't forget to remove console.logs, so others won't have to!

Reply:
will do, thank ou